docs/core/deploying/trim-self-contained.md

```diff
@@ -7,14 +7,15 @@ ms.date: 04/03/2020
 ---
 # Trim self-contained deployments and executables
 
-When publishing an application self-contained, the .NET Core runtime is bundled together with the application. This bundling adds a significant amount of content to your packaged application. When it comes to deploying your application, size is often an important factor. Keeping the size of the package application as small as possible is typically a goal for application developers.
+The globally installed runtime and framework libraries has been the most successful runtime deployment model since the inception of .NET. In this scenario, the application developer will only bundle the application and third-party assemblies with the expectation that the .NET runtime and framework will be available in the client machine. This deployment model will continue to be the dominant one in .NET Core as well but there will be other application deployment models targeted at app developers where the above model is not the best one. One such one is targeted for some application developers that want to publish a [self-contained application](runtime-patch-selection.md) where the .NET Core runtime and framework are bundled together with the application.
```

Comment:
Rephrasing and fixing grammar

Linking to base deployment overview since it hasn't been updated from runtime-dependent to framework-dependent yet

```suggestion
The [framework-dependent deployment model](index.md) has been the most successful deployment model since the inception of .NET. In this scenario, the application developer bundles only the application and third-party assemblies with the expectation that the .NET runtime and framework libraries will be available in the client machine. This deployment model continues to be the dominant one in .NET Core as well but there are some scenarios where the framework-dependent model is not optimal. The alternative is to publish a [self-contained application](index.md), where the .NET Core runtime and framework are bundled together with the application and third-party assemblies.
```

Reply:
https://github.com/dotnet/docs/pull/20165